nav2_costmap_2d/test/integration/test_collision_checker.cpp

```diff
@@ -179,6 +179,7 @@ class TestCollisionChecker : public nav2_util::LifecycleNode
     setPose(x, y, theta);
     publishFootprint();
     publishCostmap();
+    rclcpp::sleep_for(std::chrono::milliseconds(100));
```

Comment:
Can you get away with 50ms to avoid problems with the "transform tolerance" check?

Reply:
Perhaps.